doc/ocpp_201_status.md

```diff
@@ -1254,7 +1254,7 @@ This document contains the status of which OCPP 2.0.1 numbered functional requir
 | K01.FR.26 | ✅     |                                                                                                 |
 | K01.FR.27 | ✅     |                                                                                                 |
 | K01.FR.28 | ✅     |                                                                                                 |
-| K01.FR.29 |        |                                                                                                 |
+| K01.FR.29 | ✅     |                                                                                                 |
```

Comment:
Can you add a comment indicating that this FR is satisfied but not currently tested automatically?

Reply:
Or enable the test :)

Reply:
OK @drmrd you stole my line.

Reply:
TD